File: public/examples/integrations/toolkits/github/create_reply_for_review_comment_example_llm_oai.py
import os
from openai import OpenAI

USER_ID = "you@example.com"
PROMPT = "Create a reply to review comment #1 on pull request #2 in the ArcadeAI/example repository with the body 'This is a reply.'"
TOOL_NAME = "Github.CreateReplyForReviewComment"

client = OpenAI(
    base_url="https://api.arcade.dev", api_key=os.environ.get("ARCADE_API_KEY")
)

response = client.chat.completions.create(
    messages=[
        {"role": "user", "content": PROMPT},
    ],
    model="gpt-4o-mini",
    user=USER_ID,
    tools=[TOOL_NAME],
    tool_choice="generate",
)
print(response.choices[0].message.content)

File: public/examples/integrations/toolkits/github/create_reply_for_review_comment_example_llm_oai.js
import OpenAI from 'openai';

const USER_ID = 'you@example.com';
const PROMPT = "Create a reply to review comment #1 on pull request #2 in the ArcadeAI/example repository with the body 'This is a reply.'";
const TOOL_NAME = 'Github.CreateReplyForReviewComment';

const client = new OpenAI({
  baseURL: 'https://api.arcade.dev',
  apiKey: process.env.ARCADE_API_KEY
});

const response = await client.chat.completions.create({
  messages: [
    { role: 'user', content: PROMPT }
  ],
  model: 'gpt-4o-mini',
  user: USER_ID,
  tools: [TOOL_NAME],
  tool_choice: 'generate'
});

console.log(response.choices[0].message.content);
